sections/featured-product.liquid

```diff
@@ -352,17 +365,6 @@
                 {% render 'icon-with-text', block: block %}
             {%- endcase -%}
           {%- endfor -%}
-          <a
-            {% if product == blank %}
-              role="link" aria-disabled="true"
-            {% else %}
-              href="{{ product.url }}"
-            {% endif %}
-            class="link product__view-details animate-arrow"
-          >
-            {{ 'products.product.view_full_details' | t }}
-            {% render 'icon-arrow' %}
-          </a>
```

Comment:
Problem here is that because it's been moved to be now living with the `share` block, if we don't have that block enabled, the `View full details` link is gone: 
![](https://screenshot.click/21-02-xaiei-wp7fz.png)

Reply:
would it make sense for me to make the view details as a separate local block that is rendered like the other blocks?